blockstack-sdk/src/main/java/org/blockstack/android/sdk/DIDs.kt

```diff
@@ -19,6 +17,8 @@ class DIDs {
 
             if (didType == "btc-addr") {
                 return did.split(':')[2]
+            }else if (didType == "stx-addr") {
```

Comment:
We now have a proper did specification for stacks 2.0. https://github.com/jolocom/stacks.js/blob/feat/add_did_resolver_package/packages/did-resolver/docs/DID_Method_Spec.md

Reply:
I did a more generic version, and updated the validation since according to the documentation the following
`did:stack:v0:15gxXgJyT5tM5A4Cbx99nwccynHYsBouzr-3`, is correct and can existe a multiple of different transactions, names, and namespaces.